### PR TITLE
[yeelight] Added support Yeelight Ceiling 3 with Night Mode

### PR DESCRIPTION
**Pull request by DragonNP**
Support Yeelight Ceiling 3 with Night Mode

- [yeelight] Added Ceiling 3 support
- Add support for Yeelight Ceiling Light with night mode

In this update, you can add Yeelight Ceiling 3 devices with night mode. Updated README

The collected JAR file:
Yandex Disk: https://yadi.sk/d/wOYZLuKhx6N4uw
Google Drive: https://drive.google.com/open?id=1UfmFBbp6l5q3r8tmEh_xFiARkjasbXIk

### CODE REVIEW

**Comment by chiefymuc** [comment]
This is a welcome addition. I have a "ceiling" (not ceiling3) device and it also has a nightmode (moon mode) in the app, and it works through the commands, e.g. "set_power;\"on\",\"smooth\","5000",5" where the "5" is for the nightlight/moon mode. Can you add this as well? Thanks!

**Comment by DragonNP** [comment]
> This is a welcome addition. I have a "ceiling" (not ceiling3) device and it also has a nightmode (moon mode) in the app, and it works through the commands, e.g. "set_power;"on","smooth","5000",5" where the "5" is for the nightlight/moon mode. Can you add this as well? Thanks!

Thank You for your support! Ceiling 3 is a ceiling with a diameter of 480 mm. I thought that there is a ceiling with a diameter of 300 mm so I wrote Ceiling3.

File: bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightCeilingWithNightHandler.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import static org.openhab.binding.yeelight.internal.YeelightBindingConstants.COLOR_TEMPERATURE_MINIMUM;
+import static org.openhab.binding.yeelight.internal.YeelightBindingConstants.COLOR_TEMPERATURE_STEP;
+
+/**
+ * The {@link YeelightCeilingWithNightHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Pogudalov Nikita - Initial contribution
+ */
+public class YeelightCeilingWithNightHandler extends YeelightHandlerBase {
```
[the remark this answers is not in the document]
Comment (DragonNP) [reply bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightCeilingWithNightHandler.java]: Okay, thanks for your help!

**Comment by chiefymuc** [comment]
> 
> 
> > This is a welcome addition. I have a "ceiling" (not ceiling3) device and it also has a nightmode (moon mode) in the app, and it works through the commands, e.g. "set_power;"on","smooth","5000",5" where the "5" is for the nightlight/moon mode. Can you add this as well? Thanks!
> 
> Thank You for your support! Ceiling 3 is a ceiling with a diameter of 480 mm. I thought that there is a ceiling with a diameter of 300 mm so I wrote Ceiling3.

In the openHAB properties of the thing, my "model" is a "ceiling1" if that helps.

**Comment by DragonNP** [comment]
> > > This is a welcome addition. I have a "ceiling" (not ceiling3) device and it also has a nightmode (moon mode) in the app, and it works through the commands, e.g. "set_power;"on","smooth","5000",5" where the "5" is for the nightlight/moon mode. Can you add this as well? Thanks!
> > 
> > 
> > Thank You for your support! Ceiling 3 is a ceiling with a diameter of 480 mm. I thought that there is a ceiling with a diameter of 300 mm so I wrote Ceiling3.
> 
> In the openHAB properties of the thing, my "model" is a "ceiling1" if that helps.

What model of lamp do you have?

**Comment by chiefymuc** [comment]
> What model of lamp do you have?

In the Yeelight App it says "Model: yeelink.light.ceiling1". In openHAB it says also ceiling1.
I bought it here: https://www.gearbest.com/ceiling-lights/pp_596249.html

**Comment by DragonNP** [comment]
> > What model of lamp do you have?
> 
> In the Yeelight App it says "Model: yeelink.light.ceiling1". In openHAB it says also ceiling1.
> I bought it here: https://www.gearbest.com/ceiling-lights/pp_596249.html

I understood. There are two types of Ceiling Light and Ceiling Lamp. I have a Ceiling Lamp 480mm you probably have a ceiling Light at 380mm

**Comment by chiefymuc** [comment]
> I understood. There are two types of Ceiling Light and Ceiling Lamp. I have a Ceiling Lamp 480mm you probably have a ceiling Light at 380mm

I measured it, it's 320mm diameter.

**Comment by DragonNP** [comment]
New pull-request: https://github.com/openhab/openhab-addons/pull/7690

**Pull request closed without merging**